realm/realm-library/src/androidTest/kotlin/io/realm/RealmMigrationTestsKotlin.kt

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import io.realm.entities.StringAndInt
+import io.realm.entities.StringOnly
+import io.realm.exceptions.RealmMigrationNeededException
+import io.realm.rule.TestRealmConfigurationFactory
+import junit.framework.Assert.assertEquals
+import junit.framework.Assert.assertFalse
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import kotlin.test.assertFailsWith
+import kotlin.test.assertTrue
+
+@RunWith(AndroidJUnit4::class)
+class RealmMigrationTestsKotlin {
+
+    @get:Rule
+    val configFactory = TestRealmConfigurationFactory()
+
+    @Before
```

Comment:
No need for these setup/teardown methods if they are just empty.

Reply:
👍